### PR TITLE
[8.0] test: ProcessCreator_withChildern less hungry

### PR DESCRIPTION
**Pull request by fstagni**

### CODE REVIEW

File: src/DIRAC/tests/Utilities/ProcessesCreator_withChildren.py [hunk 1]
```diff
@@ -15,10 +15,10 @@ def do_sum(li):
 
 
 if __name__ == "__main__":
-    my_list = list(range(150000000))
+    my_list = list(range(15000000))
 
     pool = Pool(3)
-    pool.map(do_sum, [my_list[:50000000], my_list[50000000:100000000], my_list[100000000:]])
+    pool.map(do_sum, [my_list[:5000000], my_list[5000000:10000000], my_list[10000000:]])
```
Comment (chrisburr) [inline src/DIRAC/tests/Utilities/ProcessesCreator_withChildren.py]: How about changing it to use iterators and avoiding the issue entirely?

```python
def do_sum(args):
    sum(range(*args))
    proc_name = current_process().name
    print(proc_name)
```

and

```suggestion
    pool.map(do_sum, [(0, 50000000), (50000000, 100000000), (100000000, 150000000)])
```

**Comment by fstagni** [comment]
I will merge this one as the failures in the tests are not related.

**Comment by chrisburr** [comment]
> I will merge this one as the failures in the tests are not related.

But you only half applied my suggestion!

**Comment by fstagni** [comment]
> > I will merge this one as the failures in the tests are not related.
> 
> But you only half applied my suggestion!

oops! added in https://github.com/DIRACGrid/DIRAC/pull/6574

**Pull request merged**